main.go

```diff
@@ -110,6 +111,12 @@ func main() {
 		}),
 	)
 
+	if *enableAdminAPI {
+		// TODO: I'm not fully clear whether this is the desired admin path,
+		//  from prometheus codebase I would say so, but I will double check.
+		r.Handler("PUT", *routePrefix+"/api/v1/admin/wipe", handler.WipeMetricStore(ms, logger))
```

Comment:
The path looks kind of clunky, but mostly because we don't have any other API calls yet. However, I'm sure we'll got more of those, see #184 , so it's probably best to use this URL to be both consistent with Prometheus and able to version the API if needed.

Reply:
Yeah, I had the same feeling. OK we keep it then